todos-app/src/App/ListFooter/ListFooter.tsx

```diff
@@ -1,9 +1,11 @@
 import React, { useState } from "react";
-import { ITodo, TodoListType } from "../../DataStructure";
+import { ITodo, todoListType } from "../../DataStructure";
 import FilterList from "./FilterList/FilterList";
 
-const ListFooter: React.FC<TodoListType> = ({ todos }) => {
-  // const [todos, setTodos] = useState<ITodo[]>([]);
+type TodoListFooterProps = {
+  todos: ITodo[];
+};
+const ListFooter: React.FC<TodoListFooterProps> = ({ todos }) => {
```

Comment:
from now on, let's use a shorthand version for component declaration

```
const ListFooter = ({ todo }: Props) => {
```

Reply:
Fixed it. But I'm not completely sure that I did exactly as it should be